content/applications/inventory_and_mrp/quality/quality_management/quality_alerts.rst

```diff
@@ -1,81 +1,103 @@
+  order type card (Receipts, Delivery Orders, etc.), and then select an order. Click the
+  :guilabel:`Quality Alert` button at the top of the order to open a quality alert form in a new
+  page.
+
+  .. important::
+     This method can only be used if a quality check has been requested for the inventory order. The
+     :guilabel:`Quality Alert` button will not appear otherwise. If the button does not appear, a
+     quality alert can also be created by clicking the :guilabel:`⚙️ (gear)` icon at the top of the
+     page and selecting the :guilabel:`Quality Alert` option from the resulting menu.
+
+- Open the :menuselection:`Shop Floor` module, and then select a work center from the navigation bar
+  at the top of the page. Then, click the :guilabel:`⋮ (three vertical dots)` button at the
+  bottom-right of a work order card to open the :guilabel:`What do you want to do?` menu. Select the
+  :guilabel:`Create a Quality Alert` option from this menu to open a quality alert in a pop-up
+  window.
+
+.. note::
+   Depending on how a new quality alert form is opened, certain fields on the form may already be
+   filled in. For example, if a quality alert is created from a work order card in the *Shop Floor*
+   module, the :guilabel:`Product` and :guilabel:`Work Center` are pre-filled.
+
+After opening a new quality alert form, fill it out as follows:
+
+- Enter a short :guilabel:`Title` that summarizes the issue with the product.
+- If the quality alert is referencing a specific product or product variant, select it from the
+  :guilabel:`Product` or :guilabel:`Product Variant` drop-down menus.
+- If the product issue was discovered at a specific work center, select it from the :guilabel:`Work
+  Center` drop-down menu.
+- If the product is included in a specific picking order, select it from the :guilabel:`Picking`
+  drop-down menu.
+- In the :guilabel:`Team` field, select the quality team that is responsible for managing the
+  quality alert.
+- If a specific employee should be responsible for the quality alert, select them from the
+  :guilabel:`Responsible` drop-down menu.
+- In the :guilabel:`Tags` field, select any tags relevant to the quality alert from the drop-down
+  menu.
+- Use the :guilabel:`Root Cause` field to select the cause of the quality issue, if known.
+- Choose a :guilabel:`Priority` level by selecting a :guilabel:`⭐ (star)` number between one and
+  three. Quality alerts with higher priorities appear at the top of the *Quality Alerts* Kanban board
+  in the *Quality* app.
+
+At the bottom of the quality alert form are four tabs, which can be filled out as follows:
+
+- In the :guilabel:`Description` tab, enter a description of the quality issue.
+- If any actions should be taken to fix the affected product, enter them in the
+  :guilabel:`Corrective Actions` tab.
+- If any actions should be taken to prevent the issue from occurring again, enter them in the
+  :guilabel:`Preventive Actions` tab.
```

Comment:
Would reverse the order of these statements.
Start with the label then finish with the definition otherwise we're making people read half of a sentence before figuring out what it's for or what it means.